tools/generateOpenapi.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * OpenAPI Generator でエンジンのHTTPクライアント（/src/openapi下）を生成するスクリプト。
+ *
+ * NOTE: 生成元となるOpenAPIはソースから実行しているVoicevox Engineのものを使用する必要がある。
```

Comment:
まあここはドキュメントじゃないので何でも良いけど、ついでに
```suggestion
 * NOTE: 生成元となるOpenAPIはソースから実行しているVOICEVOX ENGINEのものを使用する必要がある。
```

Engineはそのままでも別に良かったかも。